marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.test.tsx

```diff
@@ -5,18 +5,35 @@ import ManageQuery from './manage-query';
 
 describe('Query Modal Component', () => {
 
-    test('Modal is not visible', () => {
-      const { queryByTestId } =  render(<ManageQuery />);
+  let query = [{ "savedQuery": { "id": "00529904-ed6e-4539-b7d8-eeddaf15aaa4", "name": "Lee", "description": "", "query": { "searchText": "", "entityTypeIds": ["Customer"], "selectedFacets": { "firstname": { "dataType": "xs:string", "stringValues": ["Lee"] } } }, "propertiesToDisplay": ["id"], "owner": "admin", "systemMetadata": { "createdBy": "admin", "createdDateTime": "2020-05-05T14:48:53.206721-07:00", "lastUpdatedBy": "admin", "lastUpdatedDateTime": "2020-05-05T14:48:53.206721-07:00" } } }]
 
-      expect(queryByTestId('manage-queries-modal')).toBeNull();
-    });
+  test('Modal is not visible', () => {
+    const { queryByTestId } = render(<ManageQuery />);
+    expect(queryByTestId('manage-queries-modal')).toBeNull();
+  });
 
-    test('Modal is visible', () => {
-        const { getByTestId } =  render(<ManageQuery />);    
+  test('Modal is visible', () => {
+    const { getByTestId } = render(<ManageQuery />);
+    expect(getByTestId('manage-queries-modal-icon')).toBeInTheDocument();
```

Comment:
click event will fail if the testid is not in the document. I would like to avoid assertions that dont add value to a test.
Moreover I see a patter here, which we should avoid as well.
All 3 test steps in 'Modal is visible' is duplicated in subsequent steps. We have to click on the icon, there is no way around it, but in doing so 'Modal is visible' is getting tested automatically and so we dont need a separate test doing just that.